Apps/WebClient/src/ClientApp/src/constants/entryType.ts

```diff
@@ -60,7 +59,7 @@ entryTypeMap.set(EntryType.Covid19LaboratoryOrder, {
     commentType: CommentEntryType.Covid19LaboratoryOrder,
     name: "COVID‑19 Tests",
     description:
-        "View and download your COVID‑19 test results as soon as they are available.",
+        "View and download your COVID-19 test results as soon as they are available",
```

Comment:
Nit: this removes the [non-breaking hyphen character](https://www.compart.com/en/unicode/U+2011) between COVID and 19.  The non-breaking hyphen prevents the potential of the text ever being wrapped like this:

> View and download your COVID-
> 19 test results...

(There are many places we don't use it, however.)

Reply:
oh interesting - should i add it back?